### PR TITLE
[shelly] Add support for Shelly BLU series of devices

### PR DESCRIPTION
**Pull request by markus7017**
This PR adds support for Shelly BLU Button 1 and BLU Door/Window

### CODE REVIEW

File: bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api/ShellyApiInterface.java [hunk 1]
```diff
@@ -34,108 +34,112 @@
  */
 @NonNullByDefault
 public interface ShellyApiInterface {
-    boolean isInitialized();
+    public boolean isInitialized();
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api/ShellyApiInterface.java:37]: Why did you add the `public` keyword in all interfaces? This is redundant and should be left away.

**Review by kaikreuzer — CHANGES_REQUESTED** [review]
Thanks, lgtm, just a small comment below.

File: bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api/ShellyApiInterface.java (hunk 1 above)
Comment (markus7017) [reply bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api/ShellyApiInterface.java:37]: you are right, stupid. Removed from all interface declarations
fixed

**Comment by markus7017** [comment]
@kaikreuzer changes are applied, please merge, so I could prepare the next one

**Comment by kaikreuzer** [comment]
@markus7017 Have you maybe forgotten to push your changes to your fork?

**Comment by markus7017** [comment]
@kaikreuzer anytging missing?

**Comment by kaikreuzer** [comment]
@markus7017 As mentioned [3 days ago](https://github.com/openhab/openhab-addons/pull/15031#issuecomment-1586307305), I do not see any new commits on this PR since my review.

**Comment by markus7017** [comment]
@kaikreuzer changes applied

**Comment by kaikreuzer** [comment]
@markus7017 Could you please also resolve the merge conflict?

File: bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/addon/addon.xml [hunk 2]
```diff
@@ -6,7 +6,6 @@
 	<type>binding</type>
 	<name>@text/addon.shelly.name</name>
 	<description>@text/addon.shelly.description</description>
-	<connection>local</connection>
```
Comment (jlaur) [inline bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/addon/addon.xml:9]: @markus7017 - it seems this was deleted by mistake, can you please add it again?

Comment (markus7017) [reply bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/addon/addon.xml:9]: fixed
